src/app/containers/PodcastExternalLinks/index.jsx

```diff
@@ -83,6 +83,14 @@ const PodcastExternalLinks = ({ brandTitle, links }) => {
   const hasMultipleLinks = links.length > 1;
   const firstLink = links[0];
 
+  const ExternalLinkTextLangs = {
```

Comment:
this variable could be defined outside of the component

Reply:
Done! :)

Reply:
Thanks :)